jujugui/static/gui/src/app/components/notification-list-item/_notification-list-item.scss

```diff
@@ -0,0 +1,86 @@
+.notification-list-item {
+    @extend %floating-panel;
+    box-sizing: border-box;
+    min-width: $left-panel-width;
+    padding: 0 20px;
+    color: $text-colour;
+    line-height: $environment-header-height;
+    margin-top: 10px;
+    cursor: pointer;
+    opacity: 0;
+
+    &--error {
+        background-color: $error;
+        color: $white;
+        .notification-list-item__hide .svg-icon {
+            stroke: $white;
+
+            &:hover {
+                stroke: rgba(255, 255, 255, 0.6);
+            }
+        }
+    }
+
+    &--visible {
+        opacity: 1;
+        animation-duration: 0.6s;
```

Comment:
Yep it is :) This is for the whole animation which also includes the height animation. the opacity change only lasts for 50% of the animation duration.